.github/workflows/production-hosting-deployment.yml

```diff
@@ -19,7 +19,9 @@ jobs:
         with:
           credentials_json: '${{ secrets.FIREBASE_SERVICE_ACCOUNT }}'
       - name: Install dependencies
-        run: yarn install yarn global add firebase-tools&& yarn build
+        run: yarn install && yarn global add firebase-tools
+      - name: Build static files
+        run: yarn build
       - name: Deploy hosting
-        run: firebase deploy --only hosting --project production
+        run: firebase deploy --only hosting --project develop
```

Comment:
This should still be production, no?

Reply:
ah :/ I was sloppy with copy and paste, yes thanks